packages/cli/src/commands/join.ts

```diff
@@ -174,21 +194,64 @@ packageVersion: string
         tag['x-displayName'] = tag['x-displayName'] || tag.name;
```

Comment:
It looks like you can reuse `.find...` in the above line https://github.com/Redocly/redocly-cli/pull/722/files#diff-3d0894febbd7923f278d468139041923d786a33f480741eabb8d465b9040eb4cR206
And also you can try avoiding using negative assertion.